pages/common/mediainfo.md

```diff
@@ -0,0 +1,15 @@
+# mediainfo
+
+> Display metadata from video and audio files.
+
+- Display metadata for a given file in the console:
+
+`mediainfo {{file}}`
```

Comment:
Perhaps `path/to/file.ext` might be more consistent with the style we've used on other pages?

Reply:
We usually use the `file.ext` syntax when adding the extension makes the command more clear. https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax see pt. 5